session.go

```diff
@@ -210,8 +210,7 @@ func (m *milterSession) Process(msg *Message) (Response, error) {
 
 	case CodeQuit:
 		// client requested session close
-		return nil, errCloseSession
-
+		return m.backend.Close(newModifier(m))
```

Comment:
Before `errCloseSession` was returned, now it's never returned

Reply:
Sorry about this mistake.

- What value do you prefer to return here?
- How about not return any value in `Close()`?

```
Close(m *Modifier) 
```

Reply:
Updated PR. Please help review. @emersion